storefront/pages/anmelden.tsx

```diff
@@ -1,7 +1,6 @@
-import { useRouter } from 'next/router';
-
 import Header from '../modules/layout/components/Header';
 import LoginForm from '../modules/auth/components/LoginForm';
+import { useRouter } from 'next/router';
```

Comment:
Same here

Reply:
on it.

Reply:
Still wrong order of imports

Reply:
No worries. I understand. Basically I think it's good to have autoformat with prettier enabled. We also have a prettier config file in the repo for this: https://github.com/unchainedshop/currybag-website/tree/master/storefront

But it's weird that my prettier did not do these changes. We have to investigate this sometime :)

Reply:
definitely, we can add [husky](https://www.npmjs.com/package/husky) and [lint-staged](https://www.npmjs.com/package/lint-staged) to automate this process to avoid unnecessary commit due to refactoring in the future.  :bulb: